tshirts.cpp

```diff
@@ -17,6 +17,8 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+    assert(size(38) == 'S');

```

Comment:
Why did you choose to make 38 as `S` instead of `M`?
How does that relate to a customer experience?